build-tools/packages/build-cli/src/library/markdown.ts

```diff
@@ -26,18 +27,63 @@ export function remarkHeadingLinks(): (tree: Node) => void {
 			if (node.children?.length > 0) {
 				const firstChild = node.children[0];
 				if (firstChild.type === "text") {
-					const headingValue = firstChild.value;
-					const slug = slugger.slug(headingValue);
+					const slug = slugger.slug(toString(node));
```

Comment:
Is this equivalent in practice? I don't know how the structure of `node` looks like so don't know what difference would I expect to see.

Reply:
Removed the node check since it proved unnecessary.